src/OrchardCore/OrchardCore.Mvc.Core/Startup.cs

```diff
@@ -17,6 +18,8 @@ namespace OrchardCore.Mvc
 {
     public class Startup : StartupBase
     {
+        private readonly static IMemoryCache _cache = new MemoryCache(new MemoryCacheOptions());
```

Comment:
Can't it be defined in the `RazorViewCompilationMemoryCacheProvider` class instead, looks like it's only used there.